glommio/src/sys/uring.rs

```diff
@@ -1144,6 +1147,8 @@ impl Reactor {
             IoRequirements::default(),
             notifier.eventfd_fd(),
             SourceType::Read(PollableStatus::NonPollable(DirectIo::Disabled), None),
+            Weak::new(),
```

Comment:
This looks very odd to me. We do we need a reference to the reactor at this point ?

My best guess is that it's because the stats struct is in the parking reactor. But should it be?
If it lives inside the ring reactor, not only you don't need this weak, but you have access to a lot more information like which ring saw the dispatch, what kind of operation we're dispatching, etc.

Reply:
See new changes, we no longer have weak references :+1: